index.js

```diff
@@ -0,0 +1,29 @@
+document.getElementById('hamburger').onclick = () => {
+  document.getElementById('navulitems').classList.add('hamburger');
+  document.getElementById('hamburger').classList.add('hamx');
+  document.getElementById('x').classList.remove('hamx');
+};
+
+document.getElementById('x').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
+
+document.getElementById('l1').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
+
+document.getElementById('l2').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
```

Comment:
- [ ] Kindly make sure to fix the close button (X) it should be displayed correctly on the mobile menu as suggested by the previous reviewer. Correctly it has a horizontal scroll issue. Kindly see the below video and where the button (X) should be shown in the last.

https://user-images.githubusercontent.com/70269507/210591415-0ab27b82-5a4a-471b-9f46-f41ebeee8d76.mov

Reply:
Hello @sja-thedude . I check your's and others' requests and I found it seems different in google chrome and Mozilla Firefox
.  Here are the screenshots for each margin and browser.  
![Screenshot (3)](https://user-images.githubusercontent.com/115536985/210611406-eaa6a4ab-9f82-408d-9820-89b8c4c0d3cd.png)
![Screenshot (4)](https://user-images.githubusercontent.com/115536985/210611409-b79cc894-56a4-4084-9f09-5f14508cfd27.png)
![Screenshot (5)](https://user-images.githubusercontent.com/115536985/210611412-4d65c18d-0363-4608-9578-955de6c2f00f.png)
![Screenshot (6)](https://user-images.githubusercontent.com/115536985/210611415-0aec32da-d4d6-48ee-ab07-0c675578c83c.png)
![Screenshot (7)](https://user-images.githubusercontent.com/115536985/210611417-383df438-1f51-441d-b3e2-524df96b7ec0.png)
![Screenshot (8)](https://user-images.githubusercontent.com/115536985/210611421-947b20c0-f0e0-41b9-9dea-52ea6827f2c5.png)
As you could see, I think you all check google chrome the live version but I prepared it in Mozilla, so its seen different for us. I already used my extra code review for this project. I could make it for google chrome and submit but on that time I won't have any code review left. Could you approve my project or should I write this to support?

Reply:
Hello @otmanTR, Please fill out this form [Link](https://airtable.com/shrQAqnBwek5a0O0s)